cmd/csi-snapshotter/main.go

```diff
@@ -41,9 +41,9 @@ import (
 	controller "github.com/kubernetes-csi/external-snapshotter/v2/pkg/sidecar-controller"
 	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/snapshotter"
 
-	clientset "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned"
-	snapshotscheme "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned/scheme"
-	informers "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/informers/externalversions"
+	clientset "github.com/kubernetes-csi/external-snapshotter/pkg/client/clientset/versioned"
```

Comment:
For go modules v2 and above, version has to be in the path.  See this PR:
https://github.com/kubernetes-csi/external-snapshotter/pull/240

Reply:
Also see this:
https://github.com/kubernetes-csi/external-snapshotter/blob/master/hack/README.md#update-generated-codesh

Reply:
@xing-yang thanks, I will update correspondingly